src/DynamoCoreWpf/ViewModels/PackageManager/PackagePathViewModel.cs

```diff
@@ -168,7 +187,10 @@ private void RemovePathAt(int index)
         private void CommitChanges(object param)
         {
             setting.CustomPackageFolders = new List<string>(RootLocations);
-            this.packageLoader.LoadCustomNodesAndPackages(loadPackageParams,customNodeManager);
+            if (this.packageLoader != null)
```

Comment:
if the original constructor is used, this property will be null, so check for it.